### ISSUE TITLE
Add gas projectile for veins and chem missile

### ISSUE THREAD

**Issue by pchote** [OpenRA/OpenRA#9691]
The projectile wanders randomly around the map at a slow rate until a timer expires, damaging anything that it touches.

**Comment by SMIFFGIG** [comment]
Will gas be affected by wind?
Something that wasn't properly implemented in TS.

**Comment by UberWaffe** [comment]
Sidenote: I do remember in the original TS that gas was affected by elevation. In that it easily traveled down-slope but didn't easily (or possibly at all, memory is sketchy) up-slope.

**Comment by reaperrr** [comment]
I wonder whether it would be easier to make gas clouds aircraft actors with invisible weapon and `Wanders` trait.

**Comment by pchote** [comment]
Why would you want to do that?  What is the difficulty with implementing the gas movement in a projectile?

**Comment by MustaphaTR** [comment]
Projectile? They are not Projectiles in original game, they are Particles. And @reaperrr 's isea is oretty vaild.

**Comment by pchote** [comment]
I am talking about OpenRA's projectiles, not the original game.

**Comment by MustaphaTR** [comment]
I can't see reason to make them a Projectile. Making them Actors would allow them to be more customiseable.

**Comment by pchote** [comment]
Because Projectiles are spawned by armaments, they can move around the map under their own control, and they can deal damage directly to actors.  This is all the necessary behaviour for the gas clouds, without the hacks or performance overheads required if they were done as actors.

**Comment by RE-Specto** [comment]
> Sidenote: I do remember in the original TS that gas was affected by elevation. In that it easily traveled down-slope but didn't easily (or possibly at all, memory is sketchy) up-slope.

you're right.. gas clouds tend to travel down the slope, and as well killing infantry into visceroids. if i recall correctly, they also tend to decrease in concentration with time, before disappearing completely, but it may be just the animation not sure if the damage is constant.
